wallet/build.gradle

```diff
@@ -35,7 +35,7 @@ ext {
 }
 
 android {
-    compileSdkVersion 'android-25'
+    compileSdkVersion 'android-26'
```

Comment:
We could indeed switch to the new format. Somehow the gradle plugin started with versions like this. But I'd put the switch into a different commit.